common/pom.xml

```diff
@@ -314,6 +314,10 @@
       <groupId>javolution</groupId>
       <artifactId>javolution</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hive</groupId>
+      <artifactId>hive-storage-api</artifactId>
```

Comment:
why was this added to common?

Reply:
Probably a mistake, I'll investigate and correct.